db/migrate/20230115233416_create_sidebar_sections.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateSidebarSections < ActiveRecord::Migration[7.0]
+  def change
+    create_table :sidebar_sections do |t|
+      t.integer :user_id
+      t.string :title, unique: true
```

Comment:
```suggestion
      t.string :title
```

I don't think we need `unique: true` here.